usaspending_api/etl/es_transaction_template.json

```diff
@@ -395,24 +395,10 @@
           }
         },
         "treasury_accounts": {
-          "type": "nested",
-          "properties": {
-            "aid": {"type": "keyword"},
-            "ata": {"type": "keyword"},
-            "main": {"type": "keyword"},
-            "sub": {"type": "keyword"},
-            "bpoa": {"type": "keyword"},
-            "epoa": {"type": "keyword"},
-            "a": {"type": "keyword"}
-          }
+          "type": "keyword"
         },
         "federal_accounts": {
-          "type": "nested",
-          "properties": {
-            "id": {"type": "integer"},
-            "account_title": {"type": "keyword"},
-            "federal_account_code": {"type": "keyword"}
-          }
+          "type": "keyword"
```

Comment:
Are each/either of `treasury_accounts` or `federal_accounts` expected to have their text value of their codes searchable by  the Keyword Search string?

If so, a `keyword` data type is not going to allow text analysis that would allow these to be mapped. In that case we'd need to add a 2nd field that has the compact/standard TAS / FAS symbol format stored and make that `text` data type, and make that one of the fields targeted by Keyword Search.

Reply:
Glad y'all are try fixing it making it perfect I mean y'all really are the best